src/renderer/services/neo.js

```diff
@@ -857,6 +884,7 @@ export default {
         alerts.networkException(e);
         lastClaimSent = null;
         store.commit('setGasClaim', gasClaim);
+        store.commit('setShowClaimGasModal', false);
```

Comment:
Eventually we need to move all of these store mutations out of these services. They are moving to a shared npm package to be used with the mobile/web app.